testing/set_symmetric_difference.cu

```diff
@@ -168,20 +168,20 @@ DECLARE_VARIABLE_UNITTEST(TestSetSymmetricDifferenceEquivalentRanges);
 template<typename T>
 void TestSetSymmetricDifferenceMultiset(const size_t n)
 {
-  thrust::host_vector<T> temp = unittest::random_integers<T>(2 * n);
+  thrust::host_vector<T> vec = unittest::random_integers<T>(2 * n);
```

Comment:
Why the rename?

Reply:
Another variable named `temp` is defined later in the function.